pkgs/by-name/ne/nexusmods-app/package.nix

```diff
@@ -140,10 +141,22 @@ buildDotnetModule (finalAttrs: {
       let
         runTest =
           name: script:
-          runCommand "${pname}-test-${name}" { nativeBuildInputs = [ finalAttrs.finalPackage ]; } ''
-            ${script}
-            touch $out
-          '';
+          runCommand "${pname}-test-${name}"
+            {
+              nativeBuildInputs = [ finalAttrs.finalPackage ];
+              FONTCONFIG_FILE = makeFontsConf {
+                fontDirectories = [ ];
+              };
+            }
+            ''
+              export XDG_DATA_HOME="$(pwd)/data"
```

Comment:
minor nit, but you should be able to use `$PWD` here instead of `$(pwd)`

Reply:
Yep, that works. Thanks